doc/classes/Label.xml

```diff
@@ -69,6 +69,9 @@
 		<member name="structured_text_bidi_override_options" type="Array" setter="set_structured_text_bidi_override_options" getter="get_structured_text_bidi_override_options" default="[]">
 			Set additional options for BiDi override.
 		</member>
+		<member name="tab_stops" type="PackedFloat32Array" setter="set_tab_stops" getter="get_tab_stops" default="PackedFloat32Array()">
+			Aligns text to the given tab-stops.
```

Comment:
Could an example be shown using a codeblock? How many values should I add to the array in general?

Reply:
One value for the const. width stops, or more for different width spacing (list is repeated). Not sure how to best describe it without an image.

Essentially, it's an equivalent of this list in the text editors:
<img width="753" alt="Screenshot 2023-04-16 at 22 56 18" src="https://user-images.githubusercontent.com/7645683/232338803-5db707fe-1ba7-4903-91eb-0329df52a2fa.png">